src/basics.md

```diff
@@ -1195,27 +1195,20 @@ Run the recipe with `RUST_BACKTRACE=1` to display a detailed [`backtrace`] assoc
 ## Display formatted date and time
 [![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
 
-Displays UTC in the well-known formats RFC 2822 and RFC 3339, and in a custom
-format.
+Gets and displays the current time in UTC in the well-known formats RFC 2822 and
```

Comment:
I'd suggest adding hyperlinks to key identifiers such as [`Utc::now`](https://docs.rs/chrono/*/chrono/offset/struct.Utc.html#method.now)